docs/using-the-compiler.rst

```diff
@@ -410,6 +410,9 @@ Input Description
           "engine": "chc",
           // Choose whether to output all unproved targets. The default is `false`.
           "showUnproved": true,
+          // Choose which solvers should be used, if available.
+          // See the Formal Verification section for the solvers description.
+          "solvers": ["smtlib2", "z3"],
```

Comment:
Maybe it would be better to list all available values?

Reply:
Done